{{ cookiecutter.repo_name }}/README.md

```diff
@@ -73,6 +73,44 @@ To run without dev mode it is sufficient to specify the `-f docker-compose.yml`
 option this will disable dev mode and run the project in default mode which
 only maps the root_dir into the container.
 
+## (Re)build image
+```bash
+# Set current branch name as tag
+export TAG=$(git rev-parse --abbrev-ref HEAD)
+
+# Build image tagged as current branch name
+docker build- t drtools/{{cookiecutter.repo_name}}:$TAG .
+```
+
+## Tests
+Unittests usually run on your local machine and don't depend on any services.
+
+During integration tests your local machine serves as controller. This usually
+makes debugging easier. Services must be started with docker-compose. This 
+requires you though to have the current project installed in your python env:
```

Comment:
Either missing code block or unnecessary colon at the end of last sentence. Or maybe "Integration tests" section should be level 3 (###), not level 2.